rasterio/_io.pxd

```diff
@@ -19,6 +19,13 @@ cdef class BufferedDatasetWriterBase(DatasetWriterBase):
     pass
 
 
+cdef class WarpedVRTReaderBase(DatasetReaderBase):
```

Comment:
@geowurster it's really about having a base C extension class that we'll extend with a Python class.